app/views/estimates/show.html.slim

```diff
@@ -53,12 +46,16 @@
           = render "capital_table", caption_size: "m"
 
   .govuk-button-group
-    - if @model.level_of_help == "controlled" && @model.decision == "eligible"
-      = link_to t(".continue_to_cw"), new_estimate_controlled_work_document_selection_path(params[:id]), class: "govuk-button", data: { module: "govuk-button" }
-    - else
+    - if @model.level_of_help == "certificated" || @model.decision != "eligible"
```

Comment:
question - so EL-1097 is trying to rename & re-order the CTA's on this page. Would it be best to merge that one first then come back to this?

Reply:
It could work either way - we'll either have to merge that first and rebase this, or vice versa (and it might end up being vice versa given how long it is before Will's back). But it's only a couple of lines of code so it should be fine either way.